packages/labs/analyzer/src/lib/model.ts

```diff
@@ -493,16 +496,19 @@ export interface DeprecatableDescribed extends Described {
   deprecated?: string | boolean | undefined;
 }
 
-interface LitElementDeclarationInit extends ClassDeclarationInit {
+interface CustomElementDeclarationInit extends ClassDeclarationInit {
   tagname: string | undefined;
-  reactiveProperties: Map<string, ReactiveProperty>;
   events: Map<string, Event>;
   slots: Map<string, NamedDescribed>;
   cssProperties: Map<string, NamedDescribed>;
   cssParts: Map<string, NamedDescribed>;
 }
 
-export class LitElementDeclaration extends ClassDeclaration {
+interface LitElementDeclarationInit extends CustomElementDeclarationInit {
+  reactiveProperties: Map<string, ReactiveProperty>;
```

Comment:
We'll do this in a separate PR, but I want to figure out how to reconcile `reactiveProperties` here with `ClassField`... Feels like one should just be a subtype of the other (or just a `CustomElementClassField`).